dcicutils/license_utils.py

```diff
@@ -853,6 +862,10 @@ class C4InfrastructureLicenseChecker(LicenseChecker):
             'typed-function',  # LICENSE at https://www.npmjs.com/package/typed-function?activeTab=code
 
         ],
+
+        'Other/Proprietary License': [
+            'localstack-ext'
```

Comment:
Needs a comment. Please don't add _any_ item to these lists without supporting rationale.

Also, needs a trailing comma so you don't have to change an extra line to add to the list.
```suggestion
            # This is known to be offered under Apache-2.0 license.
            # Ref: https://github.com/localstack/localstack/blob/master/LICENSE.txt
            'localstack-ext',
```